### ISSUE TITLE
Dumb it down

### ISSUE THREAD

**Issue by JuanPabloPearce** [dsync89/vita3k-batch-pkg-installer#2]
So uhhh, I am dumb as rocks with getting Python working right. I always seem to have issues setting it up and can't get it to work right (importing Pandas or installing it via Pip successfully) 

Could you dumb the installation/run process down a lot and spell it out for me?

**Comment by grimmy58** [comment]
> So uhhh, I am dumb as rocks with getting Python working right. I always seem to have issues setting it up and can't get it to work right (importing Pandas or installing it via Pip successfully)
> 
> Could you dumb the installation/run process down a lot and spell it out for me?

Hi, You're right, the author really couled of helped us dumb people out huh? I've just stumbled upon this script while trying to bulk install DLC for Vita3k (which it doesn't look like it'll do :(

That being said, I've managed by brute force and as a matter of principle to get it up and running in Windows 11 by:

Downloading Python from python.org, downloads (for me it was the latest, vanilla version of python)

I installed Python. 

I found that the script wouldn't ask me where things were and that I had to edit the 'install_pkg.py' file - I right clicked and selected 'Edit with IDLE'.

I changed the INPUT_PKG_FOLDER to the path to my folder of .pkg files - you have to double up the '\', so for every '\' in your file path make it '\\'.

I similarly changed the VITA3K_PROG_PATH to direct to my vita3k.exe.

I ran the .py and... it still didn't work - I needed to install Pandas - whatever that is.

I found a website that told me to open a Powershell window and enter "py -m pip install pandas" this worked and apparently installed pandas (shrug!)

I ran the .py again and it worked but it didn't work. I didn't have lots of games in separate, correctly named folders to enable the zRIF file lookup. My DLC .pkg files were just in one folder, and I didn't in fact need the .py to look up the zRIF files (I already had them!)

Hopefully this will help you out!

Also, hopefully the dev will add a way to point to a folder that already contains zRIFs

**Comment by ElektroStudios** [comment]
See my [Quick Start Instructions](https://github.com/dsync89/vita3k-batch-pkg-installer/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5), it may help you.

**Comment by Re-doUnderwater** [comment]
https://github.com/Re-doUnderwater/vita3k-batch-pkg-installer/tree/main

Hello i have added a revamped script with DLC and Themes support
and also a GUI .py

**Comment by dsync89** [comment]
Thanks @Re-doUnderwater , just approved and merged it! Appreciate your contribution! 💯
